### PR TITLE
New UI For Godaw

### PR DESCRIPTION
**Pull request by abhinav3967**
This is a draft PR for the New UI.

- [x] Instruments Panel
- [x] Control Panel
- [x] Song Editor
- [x] Piano Roll
- [x] Top Menu
- [x] Make UI work(Make ui work as intent)

### CODE REVIEW

File: Panels/PianoRoll/Grid.gd [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+func fix_cells(_window_size):
+	update()
+
+func set_scroll(val):
+	scroll += val
+	update()
+
+func _draw():
+	for x in range(OS.window_size.x-300):
+		draw_line(Vector2(90+x*14, 0), Vector2(90+x*14, get_node("../../../").rect_size.y),
```
Comment (Technohacker) [inline Panels/PianoRoll/Grid.gd:19]: Parent node paths aren't recommended, maybe there's a better way to do this? :thinking:

File: output.waw.wav.import [hunk 2]
```diff
@@ -0,0 +1,21 @@
+[remap]
```
Comment (Technohacker) [inline output.waw.wav.import:1]: Same here, delete since it's unnecessary :P

File: addons/test.py [hunk 3]
```diff
@@ -0,0 +1,4 @@
+for a in range(100):
```
Comment (Technohacker) [inline addons/test.py:1]: Is this script needed?

**Review by Technohacker — CHANGES_REQUESTED** [review]
These changes and a few formatting changes and it should be good to merge :D

File: Panels/PianoRoll/Grid.gd [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+func fix_cells(_window_size):
+	update()
+
+func set_scroll(val):
+	scroll += val
+	update()
+
+func _draw():
+	for x in range(OS.window_size.x-300):
```
[the remark this answers is not in the document]
Comment (abhinav3967) [reply Panels/PianoRoll/Grid.gd:18]: i'll see what i can do

**Comment by Technohacker** [comment]
LGTM, merging now :D

**Pull request merged**